packages/react/src/components/CardEditor/__snapshots__/CardEditor.story.storyshot

```diff
@@ -1035,7 +1036,7 @@ exports[`Storybook Snapshot tests and console checks Storyshots Watson IoT Exper
             >
               <button
                 className="iot--btn bx--btn bx--btn--sm bx--btn--tertiary"
-                data-testid="card-edit-form-open-editor-button"
```

Comment:
should we actually add this testid to the CardEditor so it matches?

Reply:
Done!